spec/features/hq/orphan_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.feature 'Orphan CRUD spec:', :type => :feature do
+
+  background do
+    i_sign_in_as_admin
+    an_orphan_exists 1, "Orphan 1", "Father 1", "2012-01-01", "Male", "Normal"
+    an_orphan_exists 2, "Orphan 2", "Father 2", "2011-01-01", "Female", "High"
+  end
+  
+  scenario 'There should be a list of orphans on the admin index page' do
+    visit hq_orphans_path
+    and_i_should_see "Orphan 1 Father 1"
+    and_i_should_see "Orphan 2 Father 2"
+    and_i_should_see_within "Orphan 1", "2012-01-01"
+    and_i_should_see_within "Orphan 2", "High"
+    and_i_should_see_within "Orphan 1", "Male"
+    and_i_should_see_within "Orphan 2", "Female"
+  end
+  
+  
+  scenario 'Should not be able to create new orphans directly via the UI' do
+    visit hq_orphans_path
+    expect(page).to_not have_link 'New Orphan'
+  end
+  
+  scenario 'Should be able to visit an orphan from the orphan index page' do
+    visit hq_orphans_path
+    when_i_click "Orphan 1", "Orphan 1"
+    and_i_should_be_on "hq_orphan_path", { orphan_id: 1 }
+  end
+  
+  scenario 'Should be able to edit an orphan from the orphan show page' do
+    visit_orphan( "Orphan 1" )
+    and_i_click_link "Edit Orphan"
+    and_i_should_be_on "edit_hq_orphan_path", { orphan_id: 1 }
+    expect(page).to_not have_selector '#osra_num'
+    expect(page).to have_css '#orphan_sponsorship_status[disabled]'
+    fill_in 'orphan[name]', with: 'Orphan N'
+    fill_in 'orphan[date_of_birth]', with: '2010-01-01'
+    fill_in 'orphan[father_given_name]', with: 'Father N'
+    select "Female", :from => 'orphan[gender]'
+    fill_in 'orphan[health_status]', with: 'Good'
+    check 'orphan[goes_to_school]'
+    select 'Inactive', :from => 'orphan[status]'
+    select 'High', :from => 'orphan[priority]'
+    fill_in 'orphan[family_name]', with: 'Asdf'
+    fill_in 'orphan[mother_name]', with: 'Aisha'
+    check 'orphan[mother_alive]'
+    fill_in 'orphan[father_occupation]', with: 'Merchant'
+    fill_in 'orphan[guardian_name]', with: 'Goku'
+    fill_in 'orphan[guardian_id_num]', with: '999'
+    fill_in 'orphan[guardian_name]', with: '999.999.999'
+    fill_in 'orphan[alt_contact_number]', with: '000999'
```

Comment:
`orphan[guardian_id_num] = 999` and `orphan[alt_contact_number] = 000999` are to similar
when checking if we see `999` couldn't botch match? I might be wrong.